test/instanceRegistryTest/InstanceRegistryTest.scala

```diff
@@ -0,0 +1,21 @@
+package instanceRegistryTest
+
+import org.scalatest.FlatSpec
+import org.scalatest.mockito.MockitoSugar
+import org.scalatestplus.play.components.OneAppPerSuiteWithComponents
+import play.api.mvc.{Result, Results}
+import play.api.test.{FakeRequest, Helpers}
+import play.api.test.Helpers.{GET, route}
+
+import scala.concurrent.Future
+  // the whole communication with the instance registry endpoints is about to change to use socket communication
+  
+//class InstanceRegistryTest extends FlatSpec with MockitoSugar with Results with OneAppPerSuiteWithComponents {
```

Comment:
Why is everything commented out here?

Reply:
So, for this PR: Could we remove that file?

Reply:
yes, we will reintroduce a new one testing the socket communication with the pull request corresponding to the sockets.